pkg/kubelet/pluginmanager/operationexecutor/operation_generator.go

```diff
@@ -112,11 +112,13 @@ func (og *operationGenerator) GenerateRegisterPluginFunc(
 		}
 		// We add the plugin to the actual state of world cache before calling a plugin consumer's Register handle
 		// so that if we receive a delete event during Register Plugin, we can process it as a DeRegister call.
-		actualStateOfWorldUpdater.AddPlugin(cache.PluginInfo{
+		if err = actualStateOfWorldUpdater.AddPlugin(cache.PluginInfo{
```

Comment:
Hmm, I feel like this is slightly less readable as a "one-line" assignment and conditional if statement. Should we just do:

```
err = actualState...
if err != nil {
}
```